.github/workflows/ci.yml

```diff
@@ -45,14 +45,15 @@ jobs:
       matrix:
         ruby-version: [2.3, 2.4, 2.5, 2.6, 2.7, '3.0', 3.1, 3.2, '3.3', jruby-9.4.0.0, truffleruby-head]
     steps:
+    - uses: extractions/setup-just@v2
     - uses: actions/checkout@v3
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby-version }}
     - uses: stripe/openapi/actions/stripe-mock@master
     - name: test
-      run: make ci-test
+      run: just test typecheck
```

Comment:
In the interest of having each recipe do one thing, I changed CI to just call both of these.

If we wanted to, we could define a `ci-test: test typecheck` instead, but this seemed simpler.